pkg/maps/eppolicymap/eppolicymap.go

```diff
@@ -60,6 +60,18 @@ var (
 // for testing purposes.
 func CreateWithName(mapName string) error {
 	buildMap.Do(func() {
+		EpPolicyMap = bpf.NewMap(mapName,
```

Comment:
Maybe instead of `Warning()` it should be `Fatal()`. What's the point in continuing if `fd` is 0?

Reply:
Right, it's probably useless to carry on indeed. I just changed to `Fatal()` and respined. Thanks!